### DISCUSSION TITLE
Can the eslint-webpack-plugin plugin simply reference the .eslintrc.js file?

### DISCUSSION THREAD

**Discussion by joeprivettsmart** [webpack/webpack#16857] [Q&A]
With the `eslint-webpack-plugin` plugin, it seems to want you to double-up on eslint configuration that you already have in the `.eslintrc.js` file. Why can't you just have an option to set a path to the eslint config file?

**Comment by ricardogobbosouza** [comment]
Hi @joeprivettsmart 

You can use `eslint` configuration in project root following https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats
If you want to specify the config file, use `overrideConfigFile`:

```js
// webpack.config.js
const ESLintPlugin = require('eslint-webpack-plugin');

module.exports = {
  plugins: [new ESLintPlugin({
    useEslintrc: false, // If you don't have any `.eslintrc`
    overrideConfigFile: `path_to_config_file`
  )],
};
```
